descriptors/ModuleDescriptor-template.json

```diff
@@ -592,6 +592,10 @@
     {
       "id": "bound-with-parts-storage",
       "version": "1.0"
+    },
+    {
+      "id": "authority-storage",
+      "version": "1.1"
```

Comment:
Hi @Maksat-Galymzhan, please find which module provides this interface and what is the latest version. I have suspicion that it was upgraded in the current dev cycle (Poppy release)

Reply:
you are right,  there is a new version which is [2.0](https://github.com/folio-org/mod-entities-links/blob/master/pom.xml)